frontend/src/Components/Markdown/InlineMarkdown.js

```diff
@@ -33,13 +34,28 @@ class InlineMarkdown extends Component {
       }
     }
 
+    // replace blocks in the string surrounded in backticks with <code>
+    // if the first character is a backtick then we ignore the first split and start directly with a code block
```

Comment:
```suggestion
    // Replace blocks in the string surrounded in backticks with <code> blocks
```

I think the single line is sufficient.